packages/common/driver-definitions/src/storage.ts

```diff
@@ -509,4 +509,17 @@ export interface ISnapshotFetchOptions {
 	 * container, so specifying version is not necessary for storage services.
 	 */
 	versionId?: string;
+
+	/**
+	 * List of groupId of datastores for which the snapshot needs to be fetched. If not provided, entire snapshot
+	 * will be requested from the service. It is upto the service, to include snapshot for datastores with groupIds
```

Comment:
maybe "entire snapshot" should be changed to "content with default / missing groupIDs"?